hugo/layouts/partials/card.html

```diff
@@ -0,0 +1,11 @@
+<article class="cards__element cards__element_category">
```

Comment:
according to bem, that `cards__element_category` should be `cards__element--category`